model/riscv_insts_base.sail

```diff
@@ -717,11 +717,10 @@ mapping clause encdec = MRET()
 
 function clause execute MRET() = {
   if   cur_privilege != Machine
-  then handle_illegal()
+  then { handle_illegal(); RETIRE_FAIL }
   else if ~(ext_check_xret_priv (Machine))
-  then ext_fail_xret_priv ()
-  else set_next_pc(exception_handler(cur_privilege, CTL_MRET(), PC));
-  RETIRE_FAIL
+  then { ext_fail_xret_priv(); RETIRE_FAIL }
+  else { set_next_pc(exception_handler(cur_privilege, CTL_MRET(), PC)); RETIRE_SUCCESS }
```

Comment:
Hm, might be better to not make this one a one-liner.

Reply:
Done